### PR TITLE
Add a SQL script to fetch weekly test results by job for particular tests

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: scripts/sql/test-results-by-week-per-job.sql [hunk 1]
```diff
@@ -0,0 +1,56 @@
+CREATE OR REPLACE FUNCTION pg_temp.weekly_test_results_by_job(test_substring TEXT, job_name TEXT) RETURNS TABLE (
+    name TEXT,
+    week_start TIMESTAMP WITH TIME ZONE,
+    current_successes BIGINT,
+    current_flakes BIGINT,
+    current_failures BIGINT,
+    current_runs BIGINT,
+    current_working_percentage NUMERIC
```
Comment (DennisPeriquet) [inline scripts/sql/test-results-by-week-per-job.sql:8]:
```suggestion
    current_working_percentage NUMERIC(5,2)
```

File: scripts/sql/test-results-by-week-per-job.sql [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        WHERE
+            tests.name LIKE test_substring
+            AND pj.name = job_name
+        GROUP BY
+            tests.name,
+            week_start
+    )
+    SELECT
+        *,
+        (results.current_successes + results.current_flakes) * 100.0 / NULLIF(results.current_runs, 0) AS current_working_percentage
```
Comment (DennisPeriquet) [inline scripts/sql/test-results-by-week-per-job.sql:49]:
```suggestion
        ROUND((results.current_successes + results.current_flakes) * 100.0 / NULLIF(results.current_runs, 0), 2) AS current_working_percentage
```

**Comment by DennisPeriquet** [comment]
my suggestions are optional as this looks good.

**Comment by stbenjam** [comment]
Thanks, those were good.  I've added them here

**Pull request merged**